bench.go

```diff
@@ -60,11 +132,134 @@ func (s *scaleOut) collect() error {
 
 func (s *scaleOut) createReport() (string, error) {
 	//todo @zeyuan
-	return "", nil
+	rep := &stats{Interval: int(s.t.balanceTime.Sub(s.t.addTime).Seconds())}
```

Comment:
would you like to show the report example?

Reply:
`Prev Balance leader is 50746.00, compared to origin by 0.00
Prev balance region is 63512.00, compared to origin by 0.00
Cur balance leader is 50746.00, compared to origin by 0.00
Cur balance region is 63512.00, compared to origin by 0.00
Prev latency is 0.00, compared to origin by 0.00
Cur latency is 0.00, compared to origin by 0.00`